GeneratorInterface/Pythia8Interface/interface/suep_shower.h

```diff
@@ -0,0 +1,47 @@
+#ifndef SUEPSHOWER_h
```

Comment:
we'll have to come to an agreement with GEN and O&C about whether or not this code imported from theorists needs to conform to the CMS coding rules - i'm going to leave it alone for now